docs/source/overview.md

```diff
@@ -3,18 +3,29 @@ title: Concepts Overview
+Apollo Link is designed to be a powerful way to compose actions around data handling with GraphQL. Each link represents a subset of functionality that can be composed with other links to create complex control flows of data.
+
+<div class="center"> ![Figure 1](https://i.imgur.com/YvS5Enu.png)</div>
+
+At a basic level, a link is a function that takes an operation and returns an observable. An operation is an object with the following information:
 - `query`: A `DocumentNode` (parsed GraphQL Operation) describing the operation taking place
 - `variables`: A map of variables being sent with the operation
 - `operationName`: A string name of the query if it is named, otherwise it is null
 - `extensions`: A map to store extensions data to be sent to the server
-- `getContext`: A function to return the context of the request
-- `setContext`: A function that takes either a new context object, or a function which receives the previous context and returns a new one. (Think of it like `setState` from React)
+- `getContext`: A function to return the context of the request. This context can be used by links to determine which actions to perform. 
+- `setContext`: A function that takes either a new context object, or a function which receives the previous context and returns a new one. It behaves similarly to `setState` from React.
 - `toKey`: A function to convert the current operation into a string to be used as a unique identifier
 
+We can chain these links together so that the first link operates on an operation object and each subsequent link operates on the result of the previous link. This allows us to "compose" actions and implement complex data handling logicin an elegant manner.  
+
 <h2 id="request">Request</h2>
 
-At the core of a link is the `request` method. A link's request is called every time `execute` is run on that link chain (typically every operation). The request is where the operation is given to the link to return back data of some kind. Request must return an observable. Depending on where the link is in the stack, it will either use the second parameter to a link (the next link in the chain) or return back an `ExecutionResult` on its own.
+At the core of a link is the `request` method. It takes the following arguments:
+
+- `operation`: The operation being passed through the link.
+- `forward`: (optional) Specifies the next link in the chain of links.
+
+A link's `request` method is called every time `execute` is run on that link chain, which typically occurs for every operation passed through the link chain. When the `request` method is called, the link "receives" an operation has to return back data of some kind in the form of an `Observable`. Depending on where the link is in the chain (i.e. whether or it is at the end of the chain), it will either use the `forward`, the second parameter specifying the next link in the chain, or return back an `ExecutionResult` on its own.

```

Comment:
>the link "receives" an operation has to return back data of

missing *that*?

Reply:
I think it should be 'the link "receives" an operation and has to return back data of some kind." Fixed now.